src/main/kotlin/no/fdk/fdk_reasoning_service/service/Utils.kt

```diff
@@ -142,6 +142,12 @@ fun Model.containsTriple(subj: String, pred: String, obj: String): Boolean {
     } catch (ex: Exception) { false }
 }
 
+fun Statement.hasRDFType(type: Resource): Boolean =
```

Comment:
Kunne du ha brukt jena sin boolean `boolean hasProperty(Property p, String o)`
Ref https://jena.apache.org/documentation/javadoc/jena/org/apache/jena/rdf/model/Resource.html#hasProperty(org.apache.jena.rdf.model.Property,java.lang.String)

Ref også
https://stackoverflow.com/questions/16733894/jena-how-to-check-if-an-instance-belongs-to-a-certain-class

Reply:
Tenkte det bare var for de med objektverdi av type literal, men ser de også har RDFNode som input. Jeg kan teste i morgen

Reply:
Fungerte flott det: 42e817f